lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -4163,7 +4163,6 @@ static SwiftASTContext::TypeOrDecl DeclToTypeOrDecl(swift::ASTContext *ast,
     case swift::DeclKind::Module:
     case swift::DeclKind::Missing:
     case swift::DeclKind::MissingMember:
-    case swift::DeclKind::MacroExpansion:
```

Comment:
Where did the MacroExpansion decl kind go?
It's in the list of DeclKinds as far as I can tell. 
https://github.com/apple/swift/blob/f758144b55ec7f6846f3a2653935b7c3934d2ac3/include/swift/AST/Decl.h#L127
https://github.com/apple/swift/blob/f758144b55ec7f6846f3a2653935b7c3934d2ac3/include/swift/AST/DeclNodes.def#L195

Reply:
This was a duplicate. Github conveniently left out the context, scroll up 4 lines and you'll see it's already listed.

Reply:
Ah, there it is. Thanks.